docs/run-local.md

```diff
@@ -58,13 +58,19 @@ In order to launch or update the system to the latest built images or chosen con
 #### Bash
 
 ```bash
-./launch-or-update-local.sh [OPTIONS]
+./launch-or-update-local.sh
+# or to run without getting the latest development chart versions
```

Comment:
I think we should append what that actually means, I'd suggest 
instead of: `# or to run without getting the latest development chart versions`
this: `# or to run without getting the latest development chart versions ( run with versions that are currently in prod)`

Reply:
Updated with suggested help config